runtime/near-vm-runner/src/tests/runtime_errors.rs

```diff
@@ -445,7 +445,7 @@ fn test_stack_instrumentation_protocol_upgrade() {
         .opaque_error()
         .expects(&[
             expect![[r#"
-                VMOutcome: balance 4 storage_usage 12 return data None burnt gas 6789985365 used gas 6789985365
+                VMOutcome: balance 4 storage_usage 12 return data None burnt gas 18136872021 used gas 18136872021
```

Comment:
Can you please explain why these gas values changed?

Reply:
More context [here](https://github.com/near/nearcore/pull/9364#discussion_r1276553601), but in general no, I haven’t spent and don’t think it is a good time investment to figure out why. Given the nature of this test the gas amount before and after "correct stack limit" fix being the same do not make any sense so the result after these changes is in a strictly better spot.

I can spend the time on this if you think it is important though.